presto-common/src/main/java/com/facebook/presto/common/function/SqlFunctionProperties.java

```diff
@@ -37,6 +37,7 @@ public class SqlFunctionProperties
     private final boolean legacyJsonCast;
     private final Map<String, String> extraCredentials;
     private final boolean warnOnCommonNanPatterns;
+    private final boolean legacyJsonExtract;
```

Comment:
might be more helpful for users to have the config property say what's different.  e.g. cannonicalizeJsonExtract() or something.